eng/pipelines/maintenance-packages.yml

```diff
@@ -96,7 +96,46 @@ stages:
               $(_InternalBuildArgs)
             displayName: $(_BuildJobDisplayName)
 
+- stage: pack
+  displayName: Pack
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      artifacts:
+        publish:
+          artifacts: false
+          logs: true
+          manifests: true
+      enableMicrobuild: true
+      enablePublishUsingPipelines: false
+      publishAssetsImmediately: false
+      enablePublishTestResults: false
+      enableSourceIndex: ${{ and(eq(variables._RunAsInternal, True), eq(variables['Build.SourceBranch'], 'refs/heads/main')) }}
+      workspace:
+        clean: all
+      jobs:
+      - ${{ if eq(variables._RunAsPublic, True) }}:
+        - job: linux
+          timeoutInMinutes: 60
+          container: LinuxContainer
+          pool:
+            vmImage: ubuntu-latest
+          strategy:
+            matrix:
+              debug:
+                _BuildConfig: Release
+          preSteps:
+          - checkout: self
+            clean: true
+          steps:
+          - script: eng/common/build.sh
+              --configuration $(_BuildConfig)
+              --restore
+              --pack
+              /p:IsPackable=true
```

Comment:
I think this could cause issues as it sets IsPackable globally for any project, i.e. tests and other tooling projects. This could even cause an issue in the arcade build infrastructure. I think defining a new property is better, i.e. `/p:SourceProjectIsPackable=true`. You would then want to respect that property in the repo root's Directory.Build.props file for non test projects.